v4/export/dump.go

```diff
@@ -244,6 +245,11 @@ func dumpDatabases(ctx context.Context, conf *Config, connectPool *connectionsPo
 			if err != nil {
 				return err
 			}
+			if tableDataIRArray == nil {
+				log.Info(fmt.Sprintf("Skip to dump data of table %s.%s, "+
+					"because all of its columns are generated", dbName, table.Name))
```

Comment:
Thanks for your reply! I reorganized the logic and implemented through another method. I added an integration test like the one you mentioned, instead of a unit test because it is a little difficult to use data-dog to mock. If it is necessary, I will try to add one.